torchtune/dev/rl/workers/datacollectors/__init__.py

```diff
@@ -1 +1,7 @@
-from .sync import SyncLLMCollector
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Thanks for adding these, but I think that Joe C's PR was adding them?

Reply:
it's the pre-commit hooks :P